syntax = "proto3";

package recommender;

service Recommender{
	rpc Recommend(WhichUser) returns (UserList){}
}

message WhichUser{
  string userID = 1;
  string userName = 2;
}
message UserList{
  string userID = 1;
  string userName = 2;
	repeated string recommendedID = 3;
  repeated string recommendedName = 4;
}
